src/mcts/params.cc

```diff
@@ -271,6 +274,7 @@ void SearchParams::Populate(OptionsParser* options) {
   options->Add<FloatOption>(kCpuctFactorId, 0.0f, 1000.0f) = 2.815f;
   options->Add<FloatOption>(kCpuctFactorAtRootId, 0.0f, 1000.0f) = 2.815f;
   options->Add<BoolOption>(kRootHasOwnCpuctParamsId) = true;
+  options->Add<BoolOption>(kTwoFoldDrawId) = true;
```

Comment:
I don't think this is going in default on without some positive results.

Reply:
Set to false